### PR TITLE
Wazuh release v3.13.0_7.7.1

### LINKED ISSUE #436: Wazuh Ansible release v3.13.0_7.7.1
Wazuh version: 3.13.0
Elastic version: 7.7.1

### Tasks

- [x] Adapt to new versions (3.13.0 - 7.7.1) 

- [x] Solve #435 issue (_Fetch ES template from wazuh/wazuh repository_)

- [x] Check templates and update them if needed

- [x] Update Windows MD5 hash

- [x] Update changelog

- [ ] Update documentation references

- [ ] Pass Molecule tests

- [ ] Tag: v3.13.0_7.7.1

- [ ] Draft release

### PR DESCRIPTION
**Pull request by Zenidd**
Hi team,

This PR closes #436 and #435 

Greetings,
JP

### CODE REVIEW

File: roles/elastic-stack/ansible-elasticsearch/tasks/main.yml [hunk 1]
```diff
@@ -67,9 +67,9 @@
   when:
     - elasticsearch_xpack_security
 
-- name: Configure Elasticsearch.
-  template:
-    src: elasticsearch.yml.j2
+- name: Fetch latest Wazuh alerts template
+  get_url:
+    url: https://github.com/wazuh/wazuh/blob/master/extensions/elasticsearch/7.x/wazuh-template.json
```
Comment (jmgarcia404) [inline roles/elastic-stack/ansible-elasticsearch/tasks/main.yml:72]: `master` should be a tag according to versions specified

File: roles/opendistro/opendistro-kibana/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+- name: restart kibana
+  service: name=kibana state=restarted
```
Comment (jmgarcia404) [inline roles/opendistro/opendistro-kibana/handlers/main.yml:3]: service:
  name: kibana
  state: restarted

File: roles/wazuh/ansible-filebeat-oss/handlers/main.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+- name: restart filebeat
+  service: name=filebeat state=restarted
```
Comment (jmgarcia404) [inline roles/wazuh/ansible-filebeat-oss/handlers/main.yml:3]: service:
  name: filebeat
  state: restarted

File: roles/wazuh/ansible-filebeat-oss/templates/elasticsearch.yml.j2 [hunk 4]
```diff
@@ -162,6 +162,7 @@
       "data.dstip",
```
Comment (jmgarcia404) [inline roles/wazuh/ansible-filebeat-oss/templates/elasticsearch.yml.j2:162]: This template should not exists, add the dynamic fetch for OpenDistro also please

Comment (Zenidd) [reply roles/wazuh/ansible-filebeat-oss/templates/elasticsearch.yml.j2]: Done on 600dd40896cdec05c9211731e45b7987537f1c4a.

File: roles/wazuh/ansible-filebeat-oss/handlers/main.yml (hunk 3 above)
Comment (Zenidd) [reply roles/wazuh/ansible-filebeat-oss/handlers/main.yml:3]: Done on eb276a1f731515bd114334f6e4041f36068e0558

File: roles/opendistro/opendistro-kibana/handlers/main.yml (hunk 2 above)
Comment (Zenidd) [reply roles/opendistro/opendistro-kibana/handlers/main.yml:3]: Done on eb276a1f731515bd114334f6e4041f36068e0558

File: playbooks/wazuh-manager-oss.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+- hosts: managers
+  roles:
+#    - role: ../roles/wazuh/ansible-wazuh-manager
```
[the remark this answers is not in the document]
Comment (Zenidd) [reply playbooks/wazuh-manager-oss.yml]: Done on 4c0289cdf88487e6f4c7780c0cf0f195eed0cd1d

File: playbooks/wazuh-manager-oss.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    - role: ../roles/wazuh/ansible-filebeat-oss
+      filebeat_output_elasticsearch_hosts:
+      - "172.16.0.161:9200"
```
[the remark this answers is not in the document]
Comment (Zenidd) [reply playbooks/wazuh-manager-oss.yml]: Done on 4c0289cdf88487e6f4c7780c0cf0f195eed0cd1d

File: roles/elastic-stack/ansible-elasticsearch/tasks/main.yml (hunk 1 above)
Comment (Zenidd) [reply roles/elastic-stack/ansible-elasticsearch/tasks/main.yml]: Done on d82d1b51d149ee4f1d9f9f66b2566e37ae9800de

**Pull request merged**